backend/src/layer/data/database/factory.rs

```diff
@@ -99,10 +99,10 @@ mod tests {
         let factory = DataAccessFactory::new(info, true)
             .await
             .expect("failed to access test database");
-        let _ = factory.get_command_data_access();
-        let _ = factory.get_image_review_data_access();
-        let _ = factory.get_mealplan_management_data_access();
-        let _ = factory.get_request_data_access();
+        factory.get_command_data_access();
+        factory.get_image_review_data_access();
+        factory.get_mealplan_management_data_access();
+        factory.get_request_data_access();
```

Comment:
Actually, using `let _ = ...` is the ideomatic rust way of explicitly specifying that you dont want to use something. So i consider this better than specifying `allow(unsued_must_used)`

Reply:
maybe because my clippy level is set to pedantic?

Reply:
can you provide some mor information on this please? like which clippy lint complains here?

Reply:
non-binding `let` without a type annotation
consider adding a type annotation or removing the `let` keyword
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#let_underscore_untyped
`#[warn(clippy::let_underscore_untyped)]` implied by `#[warn(clippy::pedantic)]`clippy[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B2%5D?2#file%3A%2F%2F%2Fc%253A%2FUsers%2FPeer%2FDocuments%2FUni%2FSS23%2FPSE%2FImplementierung%2FMensaApp%2Fbackend%2Fsrc%2Flayer%2Fdata%2Fdatabase%2Ffactory.rs)